OneCommunityGlobal/HGNRest
# Parthj unit tests task controller

# Description
This PR covers the unit tests for the taskController.js

## Related PRS (if any):
None

## Main changes explained:
- Requirement files have been added which describes the test scenarios
- Added a new file `taskController.spec.js` that contains all the code.

## How to test:
1. check into current branch
2. do `npm install`
3. do `npm run test taskController` 
4. All test cases must pass

## Note:
Successfully execution results may look as follows:
<img width="1032" alt="Screenshot 2024-09-11 at 1 03 01 PM" src="https://github.com/user-attachments/assets/a4b13934-02cf-40a6-9b12-2adee761ca30">
